assets/app/view/spreadsheet.rb

```diff
@@ -2,10 +2,14 @@
 
 require 'view/stock_market'
 require 'lib/color'
+require 'view/link'
+require 'lib/storage'
 
 module View
   class Spreadsheet < Snabberb::Component
     needs :game
+    needs :spreadsheet_sort_by, default: Lib::Storage['spreadsheet_sort_by'] || 'ID', store: true
```

Comment:
you're already using local storage, so there's no reason to store it here,

change these two needs to

needs sort_by default: nil
needs sort_order default: nil

Reply:
yes, so it was the `update` function that I was missing! Will get rid of the `store`